canonical-web-and-design/snapcraft.io
# Progressive releases to master

## Done

- Progressive releases merge

## Issue / Card

Fixes #2530 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that releases happen as before (progressive releases are currently disabled)
